lib/puppet_x/firewalld/property/positive_integer.rb

```diff
@@ -0,0 +1,15 @@
+module PuppetX
```

Comment:
Is it a bit overkill to create a PuppetX class for this property rather than just using a validate block in the type?  This would fall outside of environment isolation provided by puppet generate types.....

Reply:
Indeed, it is more code but `validate` is not specific to a field/property. We might want to reuse that validation for other fields and avoid code duplication